include/prereq-build.mk

```diff
@@ -53,6 +53,18 @@ $(eval $(call TestHostCommand,ncurses, \
 	Please install ncurses. (Missing libncurses.so or ncurses.h), \
 	echo 'int main(int argc, char **argv) { initscr(); return 0; }' | \
 		gcc -include ncurses.h -x c -o $(TMP_DIR)/a.out - -lncurses))
+
+$(eval $(call TestHostCommand,libelf-dev, \
+	Please install ELF dev libs and headers. (Missing libelf.so or libelf.h), \
+	echo 'int main(int argc, char **argv) { return 0; }' | \
```

Comment:
@guidosarducci I'm getting following reports occasionally:

```
fatal error: gelf.h: No such file or directory
```

and the solution is always to install `libelf-dev`, do you've any idea why this part was not merged? Was it causing some issues for example? I'm simply toying with an idea to merge it.